dataprofiler/tests/profilers/test_column_profile_compilers.py

```diff
@@ -129,10 +130,21 @@ def test_diff_primitive_compilers(self):
                      'sum': 12.0,
                      'mean': 2.0,
                      'variance': 38.666666666666664,
-                     'stddev': 3.285085839971525
-                 }
+                     'stddev': 3.285085839971525,
+                     't-test': {
+                         't-statistic': 0.4155260166386663,
+                         'conservative': {
+                             'df': 1,
+                             'p-value': 0.749287157907667
+                         },
+                         'welch': {
+                             'df': 3.6288111187629117,
```

Comment:
degrees of freedom

Reply:
should we change to `dof`? I've seen that used in numpy or scipy

Reply:
Df stands for degrees of freedom, which determines the "curvature" of the t-distribution since the population variance is unknown: https://en.wikipedia.org/wiki/Degrees_of_freedom_(statistics)